python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -231,15 +230,31 @@ def index_html_endpoint(self, request: Request):
                 """)
 
     def build_static_routes(self):
-        def next_file_response(file_path):
-            with open(file_path, encoding="utf8") as f:
-                content = f.read().replace(
-                    "BUILDTIME_ASSETPREFIX_REPLACE_ME", f"{self._app_path_prefix}"
-                )
-                return Response(content=content, media_type=guess_type(file_path)[0])
```

Comment:
woof so this was a pretty bad regression that went out in `1.4.8`

Reply:
Yeah, I knew about it at the time but I wasn't sure if anyone would notice unless the server they're hosting dagster on is geographically far away.

Reply:
this is up in the core event loop for the server so all of these synchronous open read replace operations block from concurrently processing other requests